C++/fah_to_cel_function(CB).cpp

```diff
@@ -1,9 +1,10 @@
 #include<iostream>
+#include<iomanip>
 using namespace std;
 void fahToCel(int f,int steps){
     for(;f<=300;f+=steps){
-        int c = 5*(f-32)/9;
-        cout<<f<<" "<<c<<endl;
+        float c = 5*(f-32)/9.0;
+        cout<<f<<" "<<fixed<<setprecision(2)<<c<<endl;
```

Comment:
Add the line, "Temperature in Celcius: " in print statement before the output value

Reply:
I have successfully updated the print statement. Thank you for the feedback.